polaris-react/src/components/Popover/tests/Popover.test.tsx

```diff
@@ -396,14 +397,75 @@ describe('<Popover />', () => {
 
       mountWithApp(<Test />);
 
-      expect(popoverRef).toStrictEqual({
+      expect(popoverRef).toMatchObject({
         current: {
           forceUpdatePosition: expect.anything(),
         },
       });
     });
   });
 
+  describe('close', () => {
+    it('exposes a function that closes the popover & focuses the activator by default', () => {
+      const activatorId = 'focus-target';
+      let popoverRef: React.RefObject<PopoverPublicAPI> | null = null;
```

Comment:
This pattern is what's used above to test the other function in the imperative handle